mentors/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:18.04
+
+RUN apt update && \
+    apt install -y python3.7 python3-pip && \
+    apt install -y ffmpeg
+
+ADD requirements.txt /tmp/requirements.txt
+RUN pip3 install -r /tmp/requirements.txt && \
+    rm /tmp/requirements.txt
+
+COPY recordings /app/recordings
```

Comment:
Do we want to bake videos into this docker image? I suspect it might be better to treat this docker image as the "app" and the input files get set at runtime, e.g. 

```
docker run -v $(pwd)/recordings:/app/recordings ...
```

Reply:
Yeah that makes more sense, I'll do it once I start finalizing the image